src/components/cl/doca_urom/cl_doca_urom_coll.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2020-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```
here and in other files